vaadin-dashboard-flow-parent/vaadin-dashboard-flow/src/test/java/com/vaadin/flow/component/dashboard/DashboardItemMoveTest.java

```diff
@@ -83,6 +83,18 @@ public void setDashboardNotEditable_moveWidget_orderIsNotUpdated() {
         Assert.assertEquals(expectedRootLevelNodeIds, getRootLevelNodeIds());
     }
 
+    @Test
+    public void moveWidget_noClientUpdate() {
```

Comment:
The test names are somewhat confusing currently, all of them refer to the case of handling a client event, not doing programmatic updates.